### PR TITLE
Make non-manifest based installers have similar titles and overviews

### PR DESCRIPTION
**Pull request by sdake**
Only deltas should be the base installation technology in the title and overview section of the documentation.  (looks better)

### CODE REVIEW

**Comment by sdake** [comment]
@linsun @ldemailly mind taking a look?  This just makes the documentation a little more consistent in titles.

**Comment by linsun** [comment]
LGTM, could you run a test to make sure the left index is pointed correctly to the new url?  e.g. when I click on setup-k8s-helm instruction on the left side of the navigation in istio.io.

**Comment by sdake** [comment]
sure, I'll do that now.

**Comment by sdake** [comment]
Note during the jekyll running, there is this output:
```
Running ["ScriptCheck", "ImageCheck", "HtmlCheck", "LinkCheck"] on ["./_rakesite"] on *.html... 


Found 217 links in the cache...
Adding 156 links to the cache...
Removing 59 links from the cache...
Checking 314 external links...
Ran on 221 files!


HTML-Proofer finished successfully.
```
Clicking through works as well.

Looks like broken links are checked.

Here is a screenshot of the render:
screen shot 2018-03-26 at 8 41 03 am

File: _docs/setup/kubernetes/ansible-install.md [hunk 1]
```diff
@@ -119,4 +119,4 @@ It should be noted that when Jaeger is enabled, Zipkin is disabled whether or no
 ## Adding istioctl to PATH
 
 After executing the playbook if it is desired that the `istioctl` command line tool be added to the PATH,
-search for `Add Istio to PATH` in the output and execute the commands that are outputted  
\ No newline at end of file
+search for `Add Istio to PATH` in the output and execute the commands that are outputted
```
Comment (ldemailly) [inline _docs/setup/kubernetes/ansible-install.md]: I can't see / tell what changed here between the 2 lines ?
(also is "outputted" valid?)

File: _docs/setup/kubernetes/quick-start.md [hunk 2]
```diff
@@ -183,7 +183,7 @@ export PATH=$PWD/bin:$PATH
 ```
 
 1. Install Istio's core components. Choose one of the two _**mutually exclusive**_ options below or alternately install
-   with the [Helm Chart]({{home}}/docs/setup/kubernetes/helm.html):
+   with the [Helm Chart]({{home}}/docs/setup/kubernetes/helm-install.html):
```
[the remark this answers is not in the document]
Comment (ldemailly) [reply _docs/setup/kubernetes/quick-start.md]: oh never mind you are renaming it
you need to add a redirect though, so cached results don't 404
or just leave the URL / file name unchanged ?

[the remark this answers is not in the document]
Comment (sdake) [reply _docs/setup/kubernetes/quick-start.md]: @ldemailly wasn't aware of the need for a 404.  My only concern is that ansible-install.md and helm.md are inconsistent.  I would prefer a redirect so things were consistent.  What is your preference?  If its a redirect (for consistency) - could you point me at an example?  If not, I'll un-rename it ;)

File: _docs/setup/kubernetes/ansible-install.md (hunk 1 above)
Comment (sdake) [reply _docs/setup/kubernetes/ansible-install.md]: Changed "Instructions on" to "Instructions for" in this file.  I have no idea why git changed line 122 in the diff.  I'm not sure how to revert the line 122 change.  Any ideas?

Comment (sdake) [reply _docs/setup/kubernetes/ansible-install.md]: re outputted - the Ansible doc could use a tech writer editing.  I can take that on as a separate PR if you like.  Goal of this PR is only to unify the filenames, titles, overview descriptors on the main page rendering.

File: _docs/setup/kubernetes/quick-start.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ldemailly) [reply _docs/setup/kubernetes/quick-start.md]: for instance : https://raw.githubusercontent.com/istio/istio.github.io/master/_docs/setup/kubernetes/index.md

redirect_from: "/docs/tasks/installing-istio.html"


it's important for major pages or pages that are linked externally (this one is but not much so I suppose we could break it)

**Comment by sdake** [comment]
@sebastienvas I think the transition from the old bot (if there was one on this repo) to tide has made this patch appear as though I have not signed the CLA.  could you resolve the lack of CLA?

File: _docs/setup/kubernetes/ansible-install.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Installing with Ansible
-overview: Instructions on using the included Ansible playbook to perform installation.
+title: Installion with Ansible
```
Comment (ldemailly) [inline _docs/setup/kubernetes/ansible-install.md]: typo

use an editor with spell checking

File: _docs/setup/kubernetes/helm-install.md [hunk 4]
```diff
@@ -1,16 +1,18 @@
 ---
-title: Istio Helm Chart Instructions
-overview: Instructions for the setup and configuration of Istio using the Helm package manager.
+title: Installion with Helm
+overview: Install Istio with the included Helm chart.
 
 order: 30
 
+redirect_from: /docs/setup/kubernetes/helm.html
```
Comment (ldemailly) [inline _docs/setup/kubernetes/helm-install.md]: thx. I assume you tested it

**Pull request merged**